### DISCUSSION TITLE
Possibility on using Ocelot for Java 1.7 application?

### DISCUSSION THREAD

**Discussion by swarlos6** [inspectIT/inspectit-ocelot#1307] [Q&A] [answered]
Hi there,

i wanted to know if there is a way to use the InspectIT Ocelot Java agent to monitor applications run on Java version lower than 1.8.0? In my case it would be Java 1.7.45.

Thanks for your help :)

**Comment by mc-chaos** [comment, marked as answer]
Hi @swarlos6 , 
no inspectIT needs Java8.
But we had the same problem at our beginning with inspectIT and we could use some older Version of the OpenCensus-Agent (OC is the underlaying Tracing-Framework of inspectIT). With could do Tracing on Java7. So no metrics in Prometheus/Grafana, but Traces in Jaeger could be seen.
btw: Java7 is EOL .... :-) 
Regards, Sascha
